discord/abc.py

```diff
@@ -1435,6 +1443,11 @@ async def send(
             if len(embeds) > 10:
                 raise InvalidArgument("embeds parameter must be a list of up to 10 elements")
             embeds = [embed.to_dict() for embed in embeds]
+            
+        if suppress is not None:
+            flags = MessageFlags.suppress_embeds
```

Comment:
What if `suppress` is `False`?
```suggestion
        if suppress:
            flags = MessageFlags.suppress_embeds
```